cmd/agent/app/reporter/tchannel/reporter.go

```diff
@@ -57,6 +57,7 @@ type Reporter struct {
 	peerListMgr    *peerlistmgr.PeerListManager
 	batchesMetrics map[string]batchMetrics
 	logger         *zap.Logger
+	mFactory       metrics.Factory
```

Comment:
why do we need to store it?

Reply:
Oops, this is a leftover from a previous version.